source/supported-hardware/cml-rvp.rst

```diff
@@ -0,0 +1,125 @@
+.. _comet-lake-rvp:
+
+Comet Lake CRB Boards
+-----------------------
+
+.. note:: 10th Generation Intel® Core Processor is formally known as |Comet Lake S|.
+
+Supported Boards
+^^^^^^^^^^^^^^^^^^^^^
+
+|SPN| supports **CML S RVP with PCH H, CML S RVP with PCH V, CML S RVP with H420E** platform.
+
+
+Building
+^^^^^^^^^^
+
+To build |SPN| for |CML| platform::
+
+    python BuildLoader.py build cml
+
+Replace ``cml`` with ``cmlv`` for PCH V.
+
+The output images are generated under ``Outputs`` directory.
+
+
+Stitching
+^^^^^^^^^^
+
+1. Gather |CML| IFWI firmware image
+
+  Users can either download the full IFWI image if the IFWI image release is available, or read the exising IFWI image on the board using SPI programmer.
+  This image contains additional firmware ingredients that are required to boot on |CML|.
+
+.. note::
+  ``StitchLoader.py`` currently only supports stitching with boot guard feature **disabled**.
+
+
+2. Stitch |SPN| images into downloaded BIOS image::
+
+    python Platform/CometlakeBoardPkg/Script/StitchLoader.py -i <BIOS_IMAGE_NAME> -s Outputs/cml/Stitch_Components.bin -o <SBL_IFWI_IMAGE_NAME>
+
+    where -i = Input file, -o = Output file.
+
+For example, stitching |SPN| IFWI image ``sbl_cml_ifwi.bin`` from |CML| firmware images downloaded::
+
+    python Platform/CometlakeBoardPkg/Script/StitchLoader.py -i xxxx.bin -s Outputs/cml/SlimBootloader.bin -o sbl_cml_ifwi.bin
+
+Replace ``CometlakeBoardPkg`` and ``cml`` with ``CometlakevBoardPkg`` and ``cmlv`` for PCH V.
+
+For more details on stitch tool, see :ref:`stitch-tool` on how to stitch the IFWI image with |SPN|.
+
+
+Flashing
+^^^^^^^^^
+
+Flash the generated ``sbl_cml_ifwi.bin`` or ``sbl_cmlv_ifwi.bin`` to the target board using DediProg SF100 or SF600 programmer.
+
+
+.. note:: Please check the alignment/polarity when connecting Dediprog to the board. Please power off the board before connecting the Dediprog.
+.. note:: The connector labelled **J7G1** on the target board is for DediProg.
+.. note:: Please disconnect Deidprog before powering up the board again.
+.. note:: Please set the dediprog  to flash the ifwi binary from offset 0
+
+Slimbootloader binary for capsule image
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Build |CML|::
+
+  python BuildLoader.py build cml
+
+Replace ``cml`` with ``cmlv`` for PCH V.
+
+slimbootloader binary generated in ``outputs\cml\slimbootloader.bin`` or ``outputs\cmlv\slimbootloader.bin`` can be used for generating capsule image.

```

Comment:
I think you need to provide the command to generate capsule with GenCapsuleFirmware.py.

Reply:
I've provided command for this.